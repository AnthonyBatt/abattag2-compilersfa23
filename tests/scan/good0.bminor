array
void
integer
float
char
string
boolean
