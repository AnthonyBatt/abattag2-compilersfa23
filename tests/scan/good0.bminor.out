TOKEN_ARRAY               array
TOKEN_VOID                void
TOKEN_INTEGER             integer
TOKEN_FLOAT               float
TOKEN_CHAR                char
TOKEN_STRING              string
TOKEN_BOOLEAN             boolean
